_sass/main.scss

```diff
@@ -26,6 +26,29 @@ body {
   font-weight: bold;
 }
 
+.main-cta-container {
+  @media #{$break-tablet-up} {
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    padding: 0 2rem;
+
+    h1 {
+      line-height: 42px;
```

Comment:
I was torn over adding the styles that are on page-lead here or if to add the `@media` break up at `page-lead`. Ultimately I hedged my bets and made the line height here. I feel like the container should control the height while individual class should control presentation. 

Thoughts?

Reply:
i think it's magic.